PROTOCOL.md

```diff
@@ -965,6 +966,24 @@ class SetDirtyStateAction implements Action {
 ```
 </details>
 
+### 2.6.3. ExportSvgAction
+
+The client (or the server) sends a `ExportSvgAction` to indicate that diagram, which represents the current model state, should be exported in SVG format. The action provides the diagram svg as plain string.
+
+<details open><summary>Code</summary>
+
+```typescript
+class ExportSvgAction implements ResponseAction {
```

Comment:
It seems strange to me that this is a response action if there is no matching request for it. Could you maybe explain to me (and in the description) why that is the case?